extensions_build_config.bzl

```diff
@@ -0,0 +1,9 @@
+# See https://github.com/envoyproxy/envoy/blob/master/bazel/README.md for details on how this system works.
+EXTENSIONS = {
+    "envoy.filters.http.router": "//source/extensions/filters/http/router:config",
+    "envoy.filters.http.fault": "//source/extensions/filters/http/fault:config",
+    "envoy.filters.listener.tls_inspector": "//source/extensions/filters/listener/tls_inspector:config",
+    "envoy.filters.network.http_connection_manager": "//source/extensions/filters/network/http_connection_manager:config",
+    "envoy.tracers.zipkin": "//source/extensions/tracers/zipkin:config",
+    "envoy.transport_sockets.raw_buffer": "//source/extensions/transport_sockets/raw_buffer:config",
```

Comment:
This is just my own ignorance of this, but why don't we need to include our test server filter here?

Reply:
It's a good question, that didn't occur to me before. Tests are passing, so I think it is safe to assume that "foreign" extensions don't play along with this feature. We could look into proposing a feature over at the Envoy repository if that is desired?

Reply:
(with "foreign", I meant extensions not residing in Envoy's own code base)

Reply:
My area of interest with this is partially on our own documentation around it.  If others want to add in their own test server equivalent building on what we already have, how would they do that?  But that's only tangentially related to this. So it's probably best to move on.